src/web/components/elements/TextBlockComponent.tsx

```diff
@@ -149,6 +150,16 @@ const paraStyles = css`
 		position: relative;
 		vertical-align: baseline;
 	}
+
+	[data-dcr-style='bullet'] {
```

Comment:
I'm a little unsure about this pattern for setting styles, I've not seen it before. I think we would be better off sticking to existing patterns?

If we want to communicate style we could use either semantic html (eg. style _all_ `<li>` tags the same or use react props for something more specific?

Reply:
In this case, my suggestion of this attribute selector pattern to be more explicit that they're dcr styles and to avoid clashes because we're dangerously setting inner html so don't have a prop to set but what's an 'existing pattern'? A class?

Reply:
The danger here is we've started to build our own api for styles 😨 . Once you breach these kinds of lines other people tend to saunter along behind you and do the same thing, possibly on a larger scale.

Could we perhaps edit the html text to use an `li` element and then have a prop on TextBlockComponent telling it to style these `li` tags different - as we have here (but maybe using a `before` pseudo selector 🤷 )? That was we benefit in three ways:

1. Accessible - sematic html matters
2. Scalable, this will fit in with future work in Composer
3. Easier to reason about because dots in front of li tags is a thing

Reply:
`The danger here is we've started to build our own api for style`

I mean, it's just a selector?

> prop on TextBlockComponent telling it to style these li tags different - as we have here (but maybe using a before pseudo selector 🤷 

Yeah, totally. I think though a prop doesn't meet the goal - in fact we would want Composer to convert the usage (or provide differently) of a dot directly to the element. So we *do* need to mutate the html to get to remove the dot? 

I don't think these are semantically lists though, and are distinct for what *can* be marked up as a list in Composer. Maybe we need to convert these elements to a [`footer` element ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer). 

So actually, maybe we're keen to introduce a new footnoteBlockComponent(??) element. It wouldn't make sense in standfirst though....

Reply:
> we do need to mutate the html 

Oh sure, that's a given. The weird little (ascii?)dots need to be made bigger dots which means the html needs to be edited.

Maybe there's two things here. The dots you see in standfirsts, which really are lists, and these footer notes, which are also a thing in photo essays (of course they are).

In photo essays we replace them with their won special component, called HighlightBlockComponent

![Screenshot 2021-02-25 at 17 46 10](https://user-images.githubusercontent.com/1336821/109194583-7a415f80-7791-11eb-82da-157042f6ce92.jpg)

You could use that or a variation of it?

You'd be creating a rule based on convention: That a standfirst can havve 1 or more bulletted items but the footer note can only have one